packages/ui/src/ui/libs/monaco-yql-languages/browserProblemLog.ts

```diff
@@ -0,0 +1,14 @@
+import {Toaster} from '@gravity-ui/uikit';
+import {rumLogError} from '../../rum/rum-counter';
+const toaster = new Toaster();
+
+export const browserProblemLog = (serviceName: string, e: Error) => {
```

Comment:
```ts
export async function loadWebsqlAutocomplete() {
     try {
         return await import(/* webpackChunkName: "websql-autocomplete" */ '@gravity-ui/websql-autocomplete');
     } catch(e) {
         toaster.add({
            type: 'error',
            name: `load_websql_autocomplete`,
            title: 'Autocomplete error',
            content:
                'Your browser does not support all required features, it will work in restricted mode.',
        });
        rumLogError({message: `Failed to load "websql-autocomplete"`}, e);
        return undefined;
     }
}
```